ITEMS.md

```diff
@@ -0,0 +1,147 @@
+
+| Item                                    | Type               | Notes                                                                              |
+| --------------------------------------- | ------------------ | ---------------------------------------------------------------------------------- |
+| Beach Comb                              | Accessory          | Familiar weight beachcomb buff                                                     |
+| Guzzlr tablet                           | Accessory          | Garbo uses Guzzlr to extract a bit of value from wanderers, by gaining guzzlrbucks |
+| Kremlin's Greatest Briefcase            | Accessory          | Banish, buffs, & martinis                                                          |
+| Lil' Doctor™ bag                        | Accessory          | Banish and Insta-kill                                                              |
+| mayfly bait necklace                    | Accessory          | Used to snag a few more funfunds                                                   |
+| mint-in-box Powerful Glove              | Accessory          | Monster Replacer                                                                   |
+| packaged backup camera                  | Accessory          | Copier, +Meat Drops                                                                |
+| Unopened Eight Days a Week Pill Keeper  | Accessory          | Used to double the next potion.                                                    |
+| packaged knock-off retro superhero cape | Back               | Configureable Back-Item                                                            |
+| Protonic accelerator pack               | Back               | Bust Ghosts for buffs and profit                                                   |
+| vampyric cloake pattern                 | Back               | adventures, item drops, and +meat                                                  |
+| Asdon Martin                            | Campground         | Meat/Item/Booze Drops                                                              |
+| haunted doghouse                        | Campground         | Fetches things!                                                                    |
+| packaged cold medicine cabinet          | Campground         | Two Words: Horse Paste.                                                            |
+| packet of mushroom spores               | Campground         | Mushrooms for food and booze                                                       |
+| packet of tall grass seeds              | Campground         | Drops fertilizer, which sell for decent meat gain                                  |
+| portable Mayo Clinic                    | Campground         | Additional adventures from food and increases your cholesterol.                    |
```

Comment:
Additional adventures or double effect duration from food.

Reply:
resolved in new PR